tests/tensor_search/test_add_documents.py

```diff
@@ -811,6 +811,25 @@ def test_put_documents_filtering_int(self):
             expected_ids={'789'}
         )
 
+    def test_put_document_override_non_tensor_field(self):
+        docs_ = [{"_id": "789", "Title": "Story of Alice Appleseed", "Description": "Alice grew up in Houston, Texas."}]
+        tensor_search.add_documents(config=self.config, index_name=self.index_name_1, docs=docs_, auto_refresh=True, non_tensor_fields=["Title"])
+        tensor_search.add_documents(config=self.config, index_name=self.index_name_1, docs=docs_, auto_refresh=True)
+        resp = tensor_search.get_document_by_id(config=self.config, index_name=self.index_name_1, document_id="789", show_vectors=True)        
+
+        assert len(resp[enums.TensorField.tensor_facets]) == 1
+        assert enums.embedding in resp[enums.TensorField.tensor_facets][0]["Title"]
```

Comment:
Have the correct set of tests been run? 
For the Marqo unit tests, make sure you have marqo-os running, then use `tox` in the Marqo root dir. When I run this test I get this error:
```
assert enums.embedding in resp[enums.TensorField.tensor_facets][0]["Title"]
E       AttributeError: module 'marqo.tensor_search.enums' has no attribute 'embedding'
```
Which makes sense as embedding should be referenced via TensorField: https://github.com/marqo-ai/marqo/blob/mainline/src/marqo/tensor_search/enums.py#L37

Also the structure of tensor_facets is a list of dicts. Each dict has 2 fields. The first is the field_name: field_content mapping, and the second is TensorField.embedding: vector (list of floats)